SharpXMPP.sln

```diff
@@ -19,8 +19,26 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SharpXMPP.NUnit", "SharpXMP
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "SharpXMPP.Shared", "SharpXMPP.Shared\SharpXMPP.Shared.csproj", "{8BA87246-B76C-437C-B891-A62FF976F216}"
 EndProject
+Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Xamarin", "Xamarin", "{07404161-0A62-4BE4-864F-50C04E2295BA}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpXMPP.Client", "Xamarin\SharpXMPP.Client\SharpXMPP.Client.csproj", "{85A9E08D-C7AF-4DD8-9F28-40B70D599CEB}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpXMPP.Client.Droid", "Xamarin\SharpXMPP.Client.Droid\SharpXMPP.Client.Droid.csproj", "{B5306EAB-F739-43E2-B999-059902380796}"
+EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "SharpXMPP.Client.iOS", "Xamarin\SharpXMPP.Client.iOS\SharpXMPP.Client.iOS.csproj", "{B5B67C4B-BFCB-4598-8049-89F726A42597}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Ad-Hoc|Any CPU = Ad-Hoc|Any CPU
+		Ad-Hoc|iPhone = Ad-Hoc|iPhone
+		Ad-Hoc|iPhoneSimulator = Ad-Hoc|iPhoneSimulator
+		Ad-Hoc|Mixed Platforms = Ad-Hoc|Mixed Platforms
+		Ad-Hoc|x86 = Ad-Hoc|x86
+		AppStore|Any CPU = AppStore|Any CPU
+		AppStore|iPhone = AppStore|iPhone
+		AppStore|iPhoneSimulator = AppStore|iPhoneSimulator
+		AppStore|Mixed Platforms = AppStore|Mixed Platforms
+		AppStore|x86 = AppStore|x86

```

Comment:
Do we really need all these strange configurations? At lease get rid of `Mixed Platforms`, it seems that's redundant.

Reply:
Ok